src/main/java/ui/metrics/pbChange/PBChangeCalculator.java

```diff
@@ -0,0 +1,42 @@
+package ui.metrics.pbChange;
```

Comment:
should be lowercase `pbchange` to prevent conflicts with #27

Reply:
Fixed for real